src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature

```diff
@@ -30,4 +30,17 @@ Scenario Outline: User logs in as ROLE and edits a version
       |ROLE       |version id|product id|developer id|
       |ROLE_ACB   |6118      |1742      |1816        |
       |ROLE_ADMIN |2018      |1742      |1816        |
-      |ROLE_ONC   |6118      |1742      |1816        |
\ No newline at end of file
+      |ROLE_ONC   |6118      |1742      |1816        |
+
+Scenario: Verify message is displayed in versions to merge section
+    Given I'm logged in as "ROLE_ONC"
+    When I navigate to the version edit page for version "54" of product "236" of developer "142"
+    And I click merge version link for version "54"
+    Then I see "At least one other Version must be selected to merge" in versions to merge section
+
+Scenario: Verify error message is displayed when the version field is left blank
+    Given I'm logged in as "ROLE_ONC"
+    When I navigate to the version edit page for version "54" of product "236" of developer "142"
+    And I click merge version link for version "54"
```

Comment:
These steps are supposed to be friendly to non-coders, so writing them as close to real English is preferred:
```suggestion
    And I click the merge version link for version "54"
```